practices/practice-one/src/components/Header/Header.tsx

```diff
@@ -6,10 +6,19 @@ import Button from "components/Button/Button";
 import Paragraph from "components/Paragraph/Paragraph";
 import Input from "components/Input/Input";
 import { navLinks } from "constants/navLinks";
+import Form from "components/Form/Form";
 
 import "./header.css";
-
+import { FormEvent, useState, MouseEvent } from "react";
+import Overlay from "components/Overlay/Overlay";
 function Header() {
+  const [popup, setPopup] = useState(false);
```

Comment:
popup are too generic. It should `[isShowForm, setIsShowForm]`

Reply:
I will fix it right away